### PR TITLE
Add responses

### PR DESCRIPTION
**Pull request by lisamsmith100**
by Lisa Smith
linter was complaining quite a bit here

### CODE REVIEW

**Comment by lisamsmith100** [comment]
I see what I did re: diff between literal and constructor type.  Just wanted to put that out there.

File: lib/diagnostic.js [hunk 1]
```diff
@@ -39,24 +57,35 @@ let owner = null;
...
 
 // 5. Attach a `vocalize` method to the Pet's prototype.  The `vocalize` method
 // should return the pet's vocalization attribute.
 
-/* your answer here */
+Pet.prototype.vocalize = function () {
+  let noise = this.vocalization
+  }
+  return noise
+}
```
Comment (benjimelito) [inline lib/diagnostic.js:84]: Nice! Although you don't have to define a new variable called `noise`, you can simply have your function return `this.vocalization`

File: lib/diagnostic.js [hunk 2]
```diff
@@ -39,24 +57,35 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = new Pet()
+pet.name = 'Barbra Ann'
+pet.species = 'kitty'
+pet.breed = 'Tortoiseshell'
+pet.vocalization = 'muh muh muh muh muh muh meow'
```
Comment (benjimelito) [inline lib/diagnostic.js:64]: Looks good, although you forgot to add a `vocalize` method to your object

**Comment by benjimelito** [comment]
Good work, 4/5

**Pull request closed without merging**